src/components/UI/SubmitBTN.tsx

```diff
@@ -0,0 +1,24 @@
+import { twMerge } from 'tailwind-merge';
+import Spinner from './Spinner';
+
+interface submitBTN {
+  text: string;
+  styles?: string;
+  handlesubmit: () => void;
+}
+
+const SubmitBTN: React.FC<submitBTN> = ({ text, handlesubmit, styles }) => {
+  return (
+    <button
+      className={twMerge(
+        `bg-green-500 rounded-sm py-2 px-8 flex  first:ml-3 ${styles ?? ''} `,
+      )}
+      onClick={handlesubmit}
+    >
+      <Spinner white loading />
```

Comment:
Spinner shouldn't be present always, the component needs to know when spinner should be loaded